src/client/index.js

```diff
@@ -1,17 +1,30 @@
-/* global Meteor, Mongo */
+/* global Meteor, Mongo, Package */
 
-const client = {};
+let collection = {};
 
 if (typeof Meteor !== 'undefined' && Meteor.isClient) {
-  const JoinCollection = new Mongo.Collection('PublishJoin');
+  collection = new Mongo.Collection('PublishJoin');
 
-  client.get = function get(name) {
-    const join = JoinCollection.findOne({
+  collection.get = function get(name) {
+    const join = this.findOne({
       _id: name,
     });
 
     return join && join.value;
   };
+
+  collection.has = function get(name) {
+    return !!this.findOne({
+      _id: name,
+    });
+  };
+
+  if (Package.templating) {
+    Package.templating.Template.registerHelper('getPublishedJoin', name => collection.get(name));
+    Package.templating.Template.registerHelper('hasPublishedJoin', name => collection.has(name));
+  }
 }
 
+const client = collection;
```

Comment:
This is not a good idea. Export the collection this way pollutes the API because all functions and properties of the collection is exposed to users code. I wanted to keep the API as clean and simple as possible since the first place.
If exposing the collection to user is really that needed. I suggest you make it become a property of the client object:
```
const client = {};

if (...) {
  const collection = ...;
  
  client.get = function get(name) {
     return collection.findOne(...;
  };
  client.has = ...;
  client.PublishJoinCollection = collection;
}

export default client;
```